content/v2.0/application/booting.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Booting
+order: 40
+---
+
+Hanami applications support a **prepared** state and a **booted** state.
+
+Whether your app is prepared or booted determines whether its components are **lazily loaded** on demand (prepared stated), or **eagerly loaded** up front (booted state).
+
+This difference may not sound like much, but the ability of a prepared app to load _just enough_ of the app to undertake a particular task can be a huge performance advantage when it comes to testing, worker processes and other use cases where your app doesn't need every component to perform the job at hand.
```

Comment:
Maybe add a sentence or expand the last one saying that a classic example is a rake task that migrates db or compile assets - it only needs a tiny portion of your app loaded and configured.

Reply:
Thanks - added!